python/ray/rllib/evaluation/tf_policy_graph.py

```diff
@@ -151,7 +164,7 @@ def build_compute_actions(self,
             builder.add_feed_dict({self._prev_action_input: prev_action_batch})
         if self._prev_reward_input is not None and prev_reward_batch:
             builder.add_feed_dict({self._prev_reward_input: prev_reward_batch})
-        builder.add_feed_dict({self._is_training: is_training})
+        builder.add_feed_dict({self._is_training: False})
```

Comment:
where in the code is `_is_training` True?

Reply:
https://github.com/ray-project/ray/blob/dedd3a802a972314afbe3799bfda3e8c9b098911/python/ray/rllib/evaluation/tf_policy_graph.py#L227